tox.ini

```diff
@@ -1,13 +1,14 @@
 [tox]
 minversion = 3.5.3
-envlist = py{38,39,310,311,312,313}, norewrite, pytest6
+envlist = py{38,39,310,311,312,313,314}, norewrite, pytest6
 
 [testenv]
 deps =
     coverage
     mock
     pytest-asyncio
     pytest6: pytest==6.2.5
+pip_pre = true
```

Comment:
https://github.com/pytest-dev/pytest-asyncio/issues/1025#issuecomment-2866587131

Reply:
I think we can wait a bit until this is no longer necessary?

Reply:
Yeah, totally 👍

Reply:
Support for the development versions of Python 3.14 is now available in pytest-asyncio v1.0.0 :)

Reply:
Thanks @seifertm for the ping.